README.md

```diff
@@ -285,7 +332,7 @@ $postData = new PostData([
     ]
 ]);
 ```
-**Attention**: For nested type casting to work your Docblock definition needs to be a Fully Qualified Class Name (`\App\DTOs\TagData[]` instead of `TagData[]` and an use statement at the top)
+**Attention**: Remember, for nested type casting to work, your Docblock definition needs to be a Fully Qualified Class Name (`\App\DTOs\TagData[]` instead of `TagData[]` and a use statement at the top).
```

Comment:
Btw, the example it refers to (above, not touched in this PR [yet 😄 ]) uses `… @var \TagData[]` and not `\App\…\DataData`; seems like an existing typo there and should probably be updated, too?

Reply:
Basically we need to update our example app directory structure. We can do that. 😄